### DISCUSSION TITLE
Asus Zenbook 490UA (Confirmed Test)

### DISCUSSION THREAD

**Discussion by SA-Mousavichashmi** [dominiksalvet/asus-fan-control#118] [General]
Hi.

Hope you are doing well.

I have tested asus-fan-control on my laptop and I confirm that it works properly. Also, afc-gui works.

My Spec:
Asus Zenbook 490UA (Intel 7500U)
Ubuntu 22.04

**Comment by SA-Mousavichashmi** [comment]
I have a problem about my Asus laptop fan.

 After 60C, the laptop fan spins at full speed regardless of using fan control tools. I think bios take control of fan directly after that specific temprreature. When temp reaches below 55C, fan control tools can take control of the fan again. (I figure it out through some experiment)

Do you have any idea about how I can change this temp limit or disable this? (because with this limit using fan control is pretty useless for me)

I really appreciate your help.

**Reply by dominiksalvet** [reply]
I am sorry, but I cannot figure out why this behavior occurs... Maybe I would try to play with temperatures since I can imagine that your laptop may use fewer temperatures than other models (8 temperature points is common).
